packages/svelte-cloudinary/src/helpers/getCldOgImageUrl.ts

```diff
@@ -33,7 +33,7 @@ export function getCldOgImageUrl(
 			...options,
 			format: options.format ?? 'jpg',
 			width: options.width ?? 1200,
-			height: options.height ?? 627,
+			height: options.height ?? 630,
```

Comment:
The `627` that was here before is the right one :pray:

Reply:
Oh good catch, would you like to update that? It's in [packages/docs/src/content/docs/helpers/getcldogimageurl/configuration.mdx](./packages/docs/src/content/docs/helpers/getcldogimageurl/configuration.mdx)